src/main/java/com/ionic/sdk/agent/cipher/chunk/ChunkCipherAbstract.java

```diff
@@ -112,7 +111,8 @@ final String encryptInternal(
             final String cipherTextBase64 = denormalize(cipherText.substring(cipherTextStart, cipherTextEnd));
             return decryptInternal(keyId, cipherTextBase64, decryptAttributes);
         } else {
-            throw new IonicException(SdkError.ISAGENT_INVALIDVALUE, new IOException(cipherText));
+            final int errorCode = SdkError.ISAGENT_INVALIDVALUE;
+            throw new IonicException(errorCode, new IonicException(errorCode, cipherText));
```

Comment:
Do users of the SDK have to now handle exceptions differently?

Reply:
No?

Reply:
I apologize.  I didn't read the code correctly.  It is "no".